app/models/campaign.rb

```diff
@@ -9,8 +9,12 @@
 #
 
 class Campaign < ActiveRecord::Base
-  has_and_belongs_to_many :districts
+  has_and_belongs_to_many :legislators
+  has_and_belongs_to_many :representatives, -> { house }, class_name: 'Legislator'
+  has_and_belongs_to_many :senators, -> { senate }, class_name: 'Legislator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Prefer `has_many :through` to `has_and_belongs_to_many`.